README.md

```diff
@@ -15,3 +15,4 @@ Hereth be yon rules:
 2. All Pull Requests must be accompanied by an animated gif. No exceptions.
 3. ????
 4. PROFIT!!!
+5. three sir!
```

Comment:
This edit is already in production? @mathisto

Reply:
Yes. But it won't show until he rebases. This is one of the rangers of keeping PRs open for a long time. Your trunk is still evolving and your merge to master is going to get more and more treacherous/difficult.